lib/escape.js

```diff
@@ -53,6 +53,20 @@ exports.escapeHtml = function (input) {
 };
 
 
+exports.escapeJson = function (input) {
+
+    if (!input) {
+        return '';
+    }
+
+    return input.replace(/</g, '\\u003c')
```

Comment:
I got a difference of less than 1% looking at that benchmark, I think where it's at now is pretty solid